tsdb/wal/wal.go

```diff
@@ -673,8 +673,8 @@ func (w *WAL) log(rec []byte, final bool) error {
 
 // Segments returns the range [first, n] of currently existing segments.
 // If no segments are found, first and n are -1.
-func (w *WAL) Segments() (first, last int, err error) {
```

Comment:
I think that this function should now be reordered

Reply:
@roidelapluie What do you mean by reordering? Move it within the file?

Reply:
Yes, it is in the middle of the func (w *WAL)  functions.